packages/java-packages/snippet-replacer-maven-plugin/README.md

```diff
@@ -0,0 +1,133 @@
+# Snippet Replacer
+
+`snippet-replacer-maven-plugin` allows java devs to reference actual java code in their javadoc comments. Developers running code written by the Azure team want their documentation to be useful. This plugin helps the Azure SDK team deliver on that promise by ensuring that code samples presented in doc-comments is always _actual running code_.
```

Comment:
This tool is very useful for any Java repo and if there's nothing in the implementation that is Azure SDK specific, we should make the documentation Azure SDK agnostic. 

So, it would be good to just replace "This plugin helps the Azure SDK team deliver on that promise" with "This plugin allows Java developers to ensure code samples presented in doc-comments is always _actual running code_".